src/pages/todo.jsx

```diff
@@ -1,17 +1,96 @@
+require ('../styles/todoList.css');
+
 import React from 'react';
+import ReactDOM from 'react-dom';
 import { connect } from 'react-redux';
 
 class Todolist extends React.Component {
-	render() {
-		return (
-			<div className="wrap">
-				<h1>Lista Todo</h1>
-			    <ul>
-			    	<li>item 1</li>
-			    </ul>
-			</div>
-		);
-	}
+    render() {
+        return (
+            <TodoApp/>
+        );
+    }
 };
 
-export default connect()(Todolist);
\ No newline at end of file
+let TodoBanner = React.createClass({
```

Comment:
można w ten sposób ale to jest wersja w czystym js, my korzystamy z ES6
`class Todolist extends React.Component {} `